entrypoint.sh

```diff
@@ -29,11 +29,10 @@ ROOT_OS_RELEASE="${ROOT_OS_RELEASE:-/root/etc/os-release}"
 KERNEL_SRC_DIR="${KERNEL_SRC_DIR:-/build/usr/src/linux}"
 NVIDIA_DRIVER_VERSION="${NVIDIA_DRIVER_VERSION:-390.46}"
 NVIDIA_DRIVER_MD5SUM="${NVIDIA_DRIVER_MD5SUM:-}"
-NVIDIA_DRIVER_DOWNLOAD_URL_DEFAULT="https://us.download.nvidia.com/tesla/${NVIDIA_DRIVER_VERSION}/NVIDIA-Linux-x86_64-${NVIDIA_DRIVER_VERSION}.run"
-NVIDIA_DRIVER_DOWNLOAD_URL="${NVIDIA_DRIVER_DOWNLOAD_URL:-$NVIDIA_DRIVER_DOWNLOAD_URL_DEFAULT}"
+NVIDIA_DRIVER_DOWNLOAD_URL_DEFAULT='https://storage.googleapis.com/nvidia-drivers-${NVIDIA_DRIVER_DOWNLOAD_LOCATION}-public/TESLA/NVIDIA-Linux-x86_64-'"${NVIDIA_DRIVER_VERSION}.run"
```

Comment:
ugh .. single quotes to keep `${NVIDIA_DRIVER_DOWNLOAD_LOCATION}` for getting evaluated?
I will prefer if we can just remove this. This single quote hack and the sed below is really ugly.
The function below can return the full URL directly without any need to use `sed`.